skins/shedapp.scss

```
@@ -0,0 +1,25 @@
+.shedapp {
+    --primary-color: #df2d71;
+    --bg-color: linear-gradient(
+      190deg,
+      #03b3ff 0%,
+      #38234a 22.92%,
+      #df2d71 60.42%,
+      #fb7429 100%
+    );
+    rgba(255, 255, 255, 0.5): ;
+    --text-color: #e6ffff;
+    --link-color: #e6ffff;
+    --heading-color: #e6ffff;
+    ff: ;
+    --border-color: #dad5d1;
+    --header-bg: #a2859a;
+    --block-bg: rgba(255, 255, 255, 0.5);
+  }
+  
+  .shedapp #topnav {
+    border-bottom: 3px solid #000 !important;
+    border-image-slice: 0;
+    border-image-source: rgba(255, 255, 255, 0.5)
+  }
+  
```

Your file is corrupted.You need to rearrange the indentation of your file and delete unused parts.
```suggestion
.shedapp {
  --primary-color: #df2d71;
  --bg-color: linear-gradient(190deg, #03b3ff 0%, #38234a 22.92%, #df2d71 60.42%, #fb7429 100%);
  --text-color: #e6ffff;
  --link-color: #e6ffff;
  --heading-color: #e6ffff;
  --border-color: #dad5d1;
  --header-bg: #a2859a;
  --block-bg: rgba(255, 255, 255, 0.5);
  
  #topnav {
    border-bottom: 3px solid #000 !important;
    border-image-slice: 0;
    border-image-source: rgba(255, 255, 255, 0.5);
  }
  
}
```